code-env/python/spec/requirements.txt

```diff
@@ -9,4 +9,6 @@ deskew==0.10.33
 torch==1.11.0; python_version >= '3.10'
 torch==1.9.1; python_version <= '3.9'
 easyocr==1.7.0
-packaging==21.3
\ No newline at end of file
+packaging==21.3
+python-docx==0.8.11
+pypandoc==1.11
```

Comment:
Are these ok for the various supported python versions ?

Reply:
this was tested on python 3.6, 3.7, 3.8, 3.9 and 3.10 on the following formats: `md/odt/rtf/txt/tsv/pdf/html/csv/doc/epub/docx`